packages/components/fields/date-field/src/date-field.tsx

```diff
@@ -150,21 +153,23 @@ export type TDateFieldProps = {
   /**
    * Title of the label
    */
-  title: string | ReactNode;
+  title: ReactNode;
```

Comment:
I think a `string` is a valid type for `title`, `hint` and `description`? They are all passed to the `field-label` component, where each has a type of `string | ReactNode`

Reply:
I removed it because it was showing "Union" before in storybook (cause of the pipe). `ReactNode` essentially stands for "everything react can render" and covers `string` already.